.github/workflows/release.yml

```diff
@@ -25,7 +25,7 @@ jobs:
         run: npm install
 
       - name: Release
-        run: npx semantic-release
```

Comment:
Since the npx command always installs the latest version, I modified it to use the module listed in package.json.